app/src/main/res/layout/fragment_chapter.xml

```diff
@@ -5,4 +5,6 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/white"
+    android:clickable="true"
+    android:focusable="true"
```

Comment:
You don't need any of these, remove all you have added. The main problem you are facing, i.e. when your clicks are consumed by below Fragments is due to this line in `MainActivity`:
```
fun changeFragment(fragment: Fragment, tag: String) {
    supportFragmentManager
        .beginTransaction()
        .add(R.id.container, fragment, tag)
        .addToBackStack(tag)
        .commit()
}
```
You are adding fragments on top of each other, which is not required. It is better to use `replace`, which will replace the fragment while keeping it in the stack. That means when you press back button, your current Fragment will be taken off, and then previous fragment from the stack is re-created.